src/Microsoft.VisualStudio.Composition/LazyServices.cs

```diff
@@ -40,6 +43,24 @@ internal static bool IsAnyLazyType(this Type type)
             return false;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether a type is a Lazy`1 or Lazy`2 type.
```

Comment:
What is the difference between a Lazy1 and a Lazy2 type? And _nit_ can we change to "Lazy1 or Lazy2 type, or neither"? I was confused that the return would indicate that it was a Lazy1 type or it was a Lazy2 type.

Reply:
I've updated the xml docs to make this clear. Thanks.
FYI: The Lazy`1 is the CLR type syntax for generics. A generic type has a backtick followed by the number of generic type arguments it accepts. So there are two Lazy types: one takes T and one takes both T and TMetadata.